env/work/srw_python/uti_plot_matplotlib.py

```diff
@@ -104,10 +104,44 @@ def uti_plot2d1d(self, data, x_range, y_range, xc, yc, labels, _graphs_joined=Tr
         return self._maybe_savefig(fig)
 
     #def srw_ascii_plot(fname):
-    def uti_data_file_plot(self, _fname, _read_labels=1, _e=0, _x=0, _y=0, _graphs_joined=1, _traj_report=False, _traj_axis='x'):
+    def uti_data_file_plot(self, _fname, _read_labels=1, _e=0, _x=0, _y=0, _graphs_joined=1,
+                           _multicolumn_data=False, _column_x=None, _column_y=None, #MR31102017
+                           _scale='linear', _width_pixels=None):
         #data, mode, allrange = srw_ascii_load(fname)
         #data, mode, allrange, arLabels, arUnits = _file_load(_fname, _read_labels)
-        data, mode, allrange, arLabels, arUnits = uti_plot_com.file_load(_fname, _read_labels, _traj_report, _traj_axis)
+        data, mode, allrange, arLabels, arUnits = uti_plot_com.file_load(_fname, _read_labels, _multicolumn_data) #MR31102017
+        if not _multicolumn_data:
+            data = np.array(data)
+        if mode == 3:
+            try:
```

Comment:
Why "mode == 3"? It may have values of  _enum('T','V','H','E','HV','EV','EH','EHV')
We don't seem to need FWHM printed here. E.g. it is confusing in the case of spectrum over photon energy.
I commented this part out.

Reply:
`mode=3` was for `E`, just convenience I used for myself. It's fine it's commented out for me.